content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: One Year of Excalidraw
+date: 2021-01-01
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+image: og.png
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Excalidraw started as a way to procrastinate on January 1st, 2020, and ended up being a fully fledged whiteboard product only one year later! In this post, we'll go over the most important features that made Excalidraw great at being a virtual whiteboard for sketching hand-drawn like diagrams.
+
+We are so incredibly proud to have built something that is being used by **20k weekly active people**.
+
+https://excalidraw.com/#json=6443031091740672,amxJZJxlZAlUBLADWIukFg
+
+## Tech Stack
+
+Excalidraw is fully open source, but it also stands on the shoulders of many other projects. We couldn't do it without [Rough.js](https://github.com/rough-stuff/rough), the library that gives Excalidraw its unique look, FG_Virgil the hand-written font designed by [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) and [React](https://github.com/facebook/react), of course. [TypeScript](https://github.com/microsoft/TypeScript) is also essential to tame the complexity of our codebase, giving a helping hand to new contributors (and old ones alike). Our full list of [dependencies](https://github.com/excalidraw/excalidraw/network/dependencies) is listed on GitHub.
```

Comment:
```suggestion
Excalidraw is fully open source, but it also stands on the shoulders of many other projects. We couldn't do it without [Rough.js](https://github.com/rough-stuff/rough), the library that gives Excalidraw its unique look; FG_Virgil, the hand-written font designed by [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html); [TypeScript](https://github.com/microsoft/TypeScript), to tame the complexity of our codebase, giving a helping hand to new contributors (and old ones alike); and last but not least, [React](https://github.com/facebook/react). Our full list of [dependencies](https://github.com/excalidraw/excalidraw/network/dependencies) is listed on GitHub.
```

Reintroducing the comma before `and` which is significant here, without which it adds ambiguity — did React team contribute to the Virgil font? 

Also added comma after `FG_Virgil`.


I agree it's a bit too over-commatized here, so changed the sentences a bit, and turned commas that separate each project into semicolons.